Sprint_Tests/sofia_zahoruiko.md

```diff
@@ -0,0 +1,18 @@
+1. He had a deep understanding of the layout and working with advanced technologies at the frontend.
+After a short break of several years and a deep depression, plus listening to heavy music, he has poorly understanding of the modern trends and drinks a lot.
+If he could handle depression after his wife and kid death, he wouldn't have to find a job with a relevant salary now.
+
+2. If he had a goal and adequate salary expectations, he would have already working on the remote in the USA or in Bali with the prospect of moving to the Canary Islands and getting Spanish citizenship. 
+His dream from childhood is marrying Spanish and retiring early and growing grapes on the Balearic coast. 
+
+3. If you don't know how to work piece of code written by your colleague, you would have to write console.log everywhere and check the logical sequence.
+This is a labor-intensive task. If we had a documentation for this feature or a ticket for the implementation this feature with a description of the business logic, we would have to commit fix within a couple of hours.
+
+4. I want you to not embarrassed express your thoughts in a foreign language regardless of the mistakes, that may appear couse of lack of practice and irregularity of english classes. 
+I want everyone of you to read the book in the original by the end of year, the original language is, of course, English, and enjoyed overcoming the language barrier in communication.
+
+5. Last week she had finished implementing the feature and started a new one, by now she has been implemented most of the user interface.
+Backend has been written before she started working on the project, so she already had swagger documentation of the requests and minimal description of the functionality.
+She didn't start writing unit tests yet.
```

Comment:
Last week she finished - is a definite time marker which instructs you to use Past Simple tense - to be precise you need to use the indefinite time form instead of finite as the context doesn't mean it. 
`by now she has been implemented` - by now she has implemented - as your option is the Passive form it means that she was implemented
Back end had been written before she started - you have two actions in the past and one of them preceeds the other so Past Perfect + before + Past Simple
She hasn't started writing unit tests yet. - you state the status of the action for the current moment thus you are supposed to use Present Perfect.